src/components/CardList/components/Item/Item.js

```diff
@@ -63,6 +63,7 @@ const selectedStyles = ({ theme, selected }) =>
     label: cardlist__item--selected;
 
     background: ${theme.colors.p100};
+    color: ${theme.colors.b900};
```

Comment:
Please use the primary colors instead of blue (they are blue by default):
```suggestion
    color: ${theme.colors.p900};
```